source/javascripts/stages/stage_2.rb

```diff
@@ -26,7 +26,7 @@ def code
     def instructions
       <<-INSTR
 <p>Well done!</p>
-<p>Now please make the code above work by implementing a method called `upper` that will return a given string converted to all capitals</p>
+<p>Now please make the code above work by implementing a method called `upper`. This method should accept an argument and convert the string value to be in all capitals.</p>
```

Comment:
It doesn't ask to convert the original string value. It asks to convert the string. We're changing the word return a given as people were trying to make global variables using the old syntax so I'm thinking to test this with some students to see how they handle the wording and then make further changes. None of this is going live to unaccompanied testers. It's all controlled.